### ISSUE TITLE
Switch from realm to site

### ISSUE THREAD

**Issue by arpith** [zulip/zulip-js#9]
Not sure where realm came from 😄

**Comment by faraazahmad** [comment]
Should PR #66 be merged first?

**Comment by aero31aero** [comment]
No, it can be rebased as needed after this issue has been closed. For things like realm_filters, we can use terms like custom_filters, as we use in the webapp.
